src/main/scala/de/heikoseeberger/sbtfresh/Template.scala

```diff
@@ -313,6 +313,12 @@ private object Template {
        |}
        |""".stripMargin
 
+  def shellPromptWithGit: String =
+    """|import com.typesafe.sbt.SbtGit.GitCommand
```

Comment:
If you don't use `enablePlugins(GitBranchPrompt)` on the project then you need the import when setting the prompt manually.

I considered using `enablePlugins(...)` shortly but decided against it because it would break the convention of having your prompt defined in `shell-prompt.sbt`.